tests/gold_tests/pluginTest/lua/lua_watermark.test.py

```diff
@@ -41,9 +43,16 @@
 
 ts.Disk.remap_config.AddLine(
     'map / http://127.0.0.1:{}/'.format(server.Variables.Port) +
-    ' @plugin=tslua.so @pparam={}/watermark.lua'.format(Test.TestDirectory)
+    ' @plugin=tslua.so @pparam=watermark.lua'
 )
 
+# Configure the tslua's configuration file.
+ts.Setup.Copy("watermark.lua", ts.Variables.CONFIGDIR)
+watermark_path = os.path.join(ts.Variables.CONFIGDIR, "watermark.lua")
+ts.Setup.Chown(watermark_path, "nobody", "nobody", ignore=True)
```

Comment:
I am seeing a number of these items... Why is trafficserver.ext not doing this correctly??

Reply:
These Chowns are actually not needed.